packages/squiggle-lang/src/rescript/Reducer/Reducer_MathJs/Reducer_MathJs_Parse.res

```diff
@@ -0,0 +1,158 @@
+/*
+  MathJs Nodes
+  We make MathJs Nodes strong-typed
+*/
+module AE = Reducer_Extra_Array
+module JsG = Reducer_Js_Gate
+module Rerr = Reducer_Error
+
+type reducerError = Rerr.reducerError
+
+type rec node = {
```

Comment:
I don't see why this should be `rec`

Reply:
https://rescript-lang.org/docs/manual/latest/object#combine-types you're extending or "combining" as they say with the `...` syntax just beneath in the file, but you don't need the top level one to be a `rec`.

Reply:
Just so we're clear--- your desire to use the `combine-types` feature of objects is the reason why we can't use records and gain `.` accessor syntax?

Reply:
yes, rec in type rec node is reduntant. removing...